src/App.css

```diff
@@ -0,0 +1,82 @@
+.nav {
+  height: 7vh;
+  background-color: rgb(92, 31, 31);
+  color: white;
+}
+
+.Items {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  margin: 2% 0 2% 0;
+}
+
+.Fonts {
+  display: flex;
+  gap: 55px;
+  text-decoration: none;
+  color: white;
+}
+
+.Icons {
+  display: flex;
+  text-decoration: none;
+}
+
+.InputField {
+  width: 100%;
+  height: 100%;
+}
+
+.CryptoList {
+  display: flex;
+  border-radius: 6px;
+  width: 100%;
+  height: 100%;
+  border: solid 1px black;
+  justify-content: center;
+}
+
+.Font {
+  padding: 8%;
+}
+
+.Currency {
+  display: flex;
+  justify-content: center;
+}
+
+.images {
+  width: 75%;
```

Comment:
- [ ] [Optional] I will recommend reducing the image size for a better user experience and visual appeal. 😄